### DISCUSSION TITLE
Welcome to roost Discussions!

### DISCUSSION THREAD

**Discussion by berkes** [Flockingbird/roost#51] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by berkes** [comment]
👋🏿  Hi there! 

I'm berkes (on most social media), a Rails, Ruby and Rust developer, I'm the driver of Flockingbird, both technical and concept-wise. 

Feel free to shoot me any personal questions, if you have them. And feel most free to drop questions, ideas or simply browse around. For flockingbird, you can also reach me on the fediverse at @flockingbird@fosstodon.org: for questions and ideas if you somehow don't like, can or want to engage here.

**Comment by stevefloyd** [comment]
🤠 Howdy! 

My name is [Steve Floyd](https://stevefloyd.me/). I have worked on different facets of the web for the last 20 years through my agency @axzm as well as various startup ventures. 

My core skills are at the intersection of product design & development, content strategy, technical SEO and UX / UI design. I am the first public contributor to Flockingbird, looking to improve the overall product and add value to the Fediverse as a whole. 

I am working on a decentralized / federated software foundation that supports best in class projects just like this and helps them get funding and access to mentors. 

You can find me on the Fedi at [@steve@floyds.io](https://floyds.io/@steve). Feel free to throw me a follow and reach out anytime!
